linked_list/linkedList.js

```diff
@@ -41,11 +41,11 @@ class LinkedList {
 
 	// Adds value at end of Linked List
 	addLast (value) {
-		if(this.head === null) {
+		if (this.head === null) {
 			this.addFirst(value);
 		} else {
 			let temp = this.head;
-			while(temp.next != null) {
+			while (temp.next != null) {
```

Comment:
@Monal5031 I quoted
> HappinessLint Bear is not recommending that, so I think this is okay.
That means HappinessLint Bear doesn't have problem with '!='. But I will still change it.